roles/ceph-osd/tasks/check_mandatory_vars.yml

```diff
@@ -55,6 +56,26 @@
     - not osd_auto_discovery
     - devices|length == 0
 
+- name: verify lvm_volumes have been provided
```

Comment:
add another one to make sure `osd_objectstore == 'filestore'` only? If not, we fail.